src/plugins/intel_cpu/src/nodes/transpose.cpp

```diff
@@ -37,8 +37,52 @@ bool Transpose::isSupportedOperation(const std::shared_ptr<const ov::Node>& op,
     return true;
 }
 
+class TransposeShapeInfer : public ShapeInferEmptyPads {
+public:
+    TransposeShapeInfer(const size_t& out_rank, const std::vector<size_t>& axes_vec) : m_out_rank(out_rank), m_axes_vec(axes_vec) {
+        m_outputShape = VectorDims(out_rank, 1);
```

Comment:
> Is possible to move these members initialization to `ctor` initialization list? To avoid default initialization and then copy.

Yes. It's a good idea. Have done it.